--- pywbem-0.17.6/tests/functiontest/conftest.py.orig	2020-10-05 23:09:31.000000000 +0200
+++ pywbem-0.17.6/tests/functiontest/conftest.py	2022-04-28 16:37:35.253071130 +0200
@@ -207,7 +207,7 @@ def patched_makefile(self, mode='r', buf
         else:
             timeout = self.timeout
         t.join(timeout)
-        if t.isAlive():
+        if t.is_alive():
             raise socket.timeout
 
     return self.fd
