--- pywbem-0.17.6/pywbem_mock/_wbemconnection_mock.py.orig	2020-10-05 23:09:31.000000000 +0200
+++ pywbem-0.17.6/pywbem_mock/_wbemconnection_mock.py	2022-04-28 16:52:34.495189422 +0200
@@ -40,7 +40,10 @@ try:
     from collections import Counter
 except ImportError:
     from backport_collections import Counter
-from mock import Mock
+try:
+    from mock import Mock
+except ImportError:
+    from unittest.mock import Mock
 import six
 
 # pylint: disable=ungrouped-imports
--- pywbem-0.17.6/tests/unittest/pywbem/test_cim_obj.py.orig	2020-10-05 23:09:31.000000000 +0200
+++ pywbem-0.17.6/tests/unittest/pywbem/test_cim_obj.py	2022-04-28 17:07:31.159025596 +0200
@@ -9,7 +9,10 @@ from __future__ import absolute_import,
 import sys
 import re
 from datetime import timedelta, datetime
-from mock import patch
+try:
+    from mock import patch
+except ImportError:
+    from unittest.mock import patch
 try:
     from collections import OrderedDict
 except ImportError:
--- pywbem-0.17.6/tests/unittest/pywbem/test_itermethods.py.orig	2020-10-05 23:09:31.000000000 +0200
+++ pywbem-0.17.6/tests/unittest/pywbem/test_itermethods.py	2022-04-28 17:07:54.975230679 +0200
@@ -24,7 +24,10 @@ from __future__ import absolute_import,
 import pytest
 import six
 
-from mock import Mock
+try:
+    from mock import Mock
+except ImportError:
+    from unittest.mock import Mock
 
 # pylint: disable=wrong-import-position, wrong-import-order, invalid-name
 from ...utils import import_installed
--- pywbem-0.17.6/tests/unittest/pywbem/test_valuemapping.py.orig	2020-10-05 23:09:31.000000000 +0200
+++ pywbem-0.17.6/tests/unittest/pywbem/test_valuemapping.py	2022-04-28 17:08:21.532885981 +0200
@@ -7,7 +7,10 @@ from __future__ import absolute_import
 
 import re
 import pytest
-from mock import Mock
+try:
+    from mock import Mock
+except ImportError:
+    from unittest.mock import Mock
 
 # pylint: disable=wrong-import-position, wrong-import-order, invalid-name
 from ...utils import import_installed
--- pywbem-0.17.6/requirements.txt.orig	2020-10-05 23:09:31.000000000 +0200
+++ pywbem-0.17.6/requirements.txt	2022-04-28 17:21:40.728745549 +0200
@@ -13,7 +13,6 @@
 M2Crypto>=0.31.0; python_version < '3.0' and sys_platform != 'win32'
 mock>=2.0.0,<3.0.0; python_version == '2.6'
 mock>=2.0.0,<4.0.0; python_version >= '2.7' and python_version <= '3.5'
-mock>=2.0.0; python_version >= '3.6'
 ordereddict>=1.1; python_version == '2.6'
 ply>=3.10
 # PyYAML 5.3 has removed support for Python 3.4; fixes narrow build error
